sections/api-security.md

```diff
@@ -33,11 +35,11 @@ Depending on the security classification you may be required to establish the fo
 - Basic or Digest authentication **SHOULD NOT**  be used.
 - The `Authorization: Bearer` header **MUST** be used for authentication/authorization e.g. using a JWT token.
 - A refresh token **SHOULD** be provided for extending expiry time of existing token without having to provide the credentials again. 
-- Always set a reasonable expiration date for tokens. JWT token lifetime **SHOULD NOT*** exceed 5 minutes.
+- Always set a reasonable expiration date for tokens. An OIDC access token lifetime **SHOULD NOT*** exceed 5 minutes.
 - JWT refresh tokens **SHOULD** be used when new JWT tokens are required outside of this lifetime. 
-- All APIs **MUST** have a policy that only allows access based on a valid API key.
-- API keys **MUST** be used for client authentication. Use of API keys should only be permitted when TLS is enabled. Rotation policy for API Key should be implemented as well.
-- API keys **SHOULD NOT** be included in the URL or query string. API keys **SHOULD** be included in the HTTP header as query strings may be saved by the client or server in unencrypted format by the browser or server application. 
+- All APIs **MUST** have a policy that only allows access to an authenticated client application.
+- Client application identity **MUST** be established via a consistent mechanism. This may be via an API Key, or via a more robust mechanism such as an OAuth 2.0 asserted server identity. Use of API keys should only be permitted when TLS is enabled. Rotation policy for API Key/secret should be implemented where applicable.
```

Comment:
This goes against principles that suggest the mechanism only needs to be strong enough to ensure the relevant security of the client and of the agency moving any data. If the data is not classified, then it seems unreasonable to require an API key for every application. Open APIs for public data for which there would be no restrictions on who can get an API key, or who could see the information should not require an API key to be provided, as this becomes somewhat invasive.  Would recommend changing this to MUST only when there is an information classification level above unclassified.  Does anyone have any other thoughts about this?

Reply:
I don't disagree - I have reworded from the perspective of API Key being an unacceptably weak mechanism for client authentication, but suitable for client identification. If SLAs are to apply to public APIs, a client identification mechanism is required, so I would suggest that a SHOULD will apply at the very least. I will reword.

Reply:
SHOULD would be OK.